test/test_quantized.py

```diff
@@ -1136,9 +1135,9 @@ def test_qconv_unpack(self, X, strideH, strideW, padH, padW, channelwise):
         strides = [strideH, strideW]
         paddings = [padH, padW]
         dilations = [1, 1]
-        W_packed = qconv_prepack(W_q, strides, paddings, dilations, groups)
+        W_packed = qconv_prepack(W_q, None, strides, paddings, dilations, groups)
         # Unpack weights weight unpacking operator (Used for serialization)
-        W_unpacked = qconv_unpack(W_packed)
+        W_unpacked = qconv_unpack(W_packed)[0]
```

Comment:
Please add a check to ensure that the unpacked bias is identical to the original bias.